### PR TITLE
Issue 8

### PR DESCRIPTION
**Pull request by KhalilMrini**
Changed Home.jsx, Body.jsx, AccountsUIWrapper.jsx, messages.js, accounts-config.js, client/main.jsx, server/main.js

### CODE REVIEW

**Comment by lfaucon** [comment]
@KhalilMrini The branch has conflicts, please fix them before I can merge.

You can do for example: 
```
git checkout develop
git pull
git checkout issue-8
git merge develop
*solve conflicts*
git commit -a
git push
```

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-*.DS_Store
```
Comment (lfaucon) [inline .gitignore]: Why did you remove this line ?

File: frog/imports/api/activities.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 import { Mongo } from 'meteor/mongo';
  
-export const Activities = new Mongo.Collection('activities');
+export const Activities = new Mongo.Collection('activities2');
```
Comment (lfaucon) [inline frog/imports/api/activities.js]: Why did you change the name of the collection ?

File: frog/imports/ui/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+export default class Home extends Component {
+
+  renderUsers(){
+    return (
+      this.props.presentUsers ? 
+      this.props.presentUsers
+        .map((user)=> user.userId)
+        .map((userId) =>
+          <li key={userId}>{Meteor.users.findOne({_id: userId}).username}</li>
```
Comment (lfaucon) [inline frog/imports/ui/Home.jsx]: When a user opens two windows, it's id will be twice in the list. The display is correct as the `<li></li>`will have the same key, so only one will be shown. However it raises a warning (see image), which we want to avoid. You can easily filter your list to keep only unique userId.
screen shot 2016-10-13 at 15 37 15

File: frog/imports/ui/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    if (text!= ""){
+      Meteor.call('messages.insert', text);
+    }
+  }
+
+  renderMessages(){
+    return (
+      this.props.messages ?
+      this.props.messages.map((msg) => (
+        <li key={msg.createdAt}><b>{msg.username} on {msg.createdAt.toUTCString()}:</b> {msg.text}</li>))
```
Comment (lfaucon) [inline frog/imports/ui/Home.jsx]: Will there be a key collision issue if two messages are created at the same second ? Maybe a safer key would be the concatenation of `username` and `createdAt`.

File: frog/imports/ui/Home.jsx (hunk 3 above)
Comment (KhalilMrini) [inline frog/imports/ui/Home.jsx]: Corrected, no more warning appears

File: frog/imports/api/activities.js (hunk 2 above)
Comment (KhalilMrini) [inline frog/imports/api/activities.js]: Corrected, was mistake when trying to test

File: .gitignore (hunk 1 above)
Comment (KhalilMrini) [inline .gitignore]: I do not recall explicitly modifying it?

Comment (lfaucon) [inline .gitignore]: I changed it back then!
